exporter/cat1_view_helper.go

```diff
@@ -87,16 +87,16 @@ func entryInfosForPatient(patient models.Record, measures []models.Measure, vsMa
 		for i, entrySection := range entrySections {
 			if entrySection != nil {
 				entry := entrySections[i].GetEntry()
-				SetCodeDisplaysForEntry(entry)
+				SetCodeDisplaysForEntry(entry, mappedDataCriteria)
 			}
 		}
 		entryInfos = appendEntryInfos(entryInfos, entrySections, mappedDataCriteria)
 	}
 	return entryInfos
 }
 
-func SetCodeDisplaysForEntry(e *models.Entry) {
-	codeDisplays := codeDisplayForQrdaOid(HqmfToQrdaOid(e.Oid))
+func SetCodeDisplaysForEntry(e *models.Entry, mapDataCriteria mdc) {
```

Comment:
I think it made more sense for symmetry for SetCodeDisplays (and most other spots that call HQMF to QRDA) to have it be passed an entry and DC, rather than an entry and piece of a DC.

Then, later, HqmfToQrdaOid is now passed in a string for VsOid alongside the HqmfOid string.

So, in short, I wanted to make any changes to other functions and calls be congruent with what's already there.